package.json

```diff
@@ -90,6 +91,9 @@
       "optional": true
     }
   },
+  "overrides": {
+    "rollup": "^4.18.0"
+  },
```

Comment:
When I brought in `@testing-library/react` multiple rollup versions appeared, this is pinned to the same version that the project depends on to avoid duplicate rollup pacakges.